content/automate/ManageIQ/Transformation/TransformationHosts/StateMachines/TransformationHostRole.class/check.yaml

```diff
@@ -0,0 +1,15 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: Check
+    inherits: 
+    description: 
+  fields:
+  - State1:
+      value: "/Transformation/TransformationHost/${#transformation_host_type}/RoleCheck"
+      on_error: "/Transformation/TransformationHost/${#transformation_host_type}/MarkAsDisabled"
```

Comment:
@fdupont-redhat 
We dont support paths in on_error/on_exit/on_entry slots?

Reply:
Well, a typo while copying. It should be `/Transformation/TransformationHost/${#transformation_host_type}.MarkAsDisabled`.